mock_service/lib/action.py

```diff
@@ -124,5 +126,5 @@ def execute(
             self._resolved_value = self._target_value_resolver(
                 action_context, self._value
             )
-
-        action_context.datapoint.set_value(self._resolved_value)
+        if self._resolved_value is not None:
+            action_context.datapoint.set_value(self._resolved_value)
```

Comment:
Missing empty line. Did you run pre-commit?

Reply:
Ah n, did not run pre-commit :)